apps/public-docsite-v9/src/Concepts/Migration/FromV8/Troubleshooting.stories.mdx

```diff
@@ -43,6 +43,11 @@ function App() {
 }
 ```
 
+## "I tried uing a layered v8 component like Callout in a layered v9 component like Dialog and my Callout appears beneath my Dialog. What's going on?"
+
+Both v9 and v8 layers set a `z-index: 10000` by default, which means the document order will resolve their z-positioning. This can lead to inconsistent behavior because z-positioning depends on the order in which elements are opened.
+To work around this set the `z-index` for one of the layers differently to ensure the correct layering. For example, for a v9 `Dialog` that contains a v8 `Callout` you can set the `Dialog`'s `z-index` to `9999` to ensure it is always beneath the `Callout`.
```

Comment:
I think the actual number used is `1000000`.

Also, I think it would be good to state that v8 `Layer` hosts tend to linger in the DOM, which can add to inconsistency.

```suggestion
Both v9 and v8 layers set a `z-index: 1000000` by default, which means the document order will resolve their z-positioning. This can lead to inconsistent behavior because z-positioning depends on the order in which elements appear in the DOM. Moreover, v8 `Layer` hosts have a tendency to linger in the DOM even after the element they are hosting is closed, which can add to this inconsistency.
To work around this, set the `z-index` for one of the layers differently to ensure the correct layering. For example, for a v9 `Dialog` that contains a v8 `Callout` you can set the `mountNode` for the `DialogSurface`'s `z-index` to `999999` to ensure it is always beneath the `Callout`.
```

Reply:
Rolled this change in separately since I've also reworked the docs based on `PortalMountNodeProvider`.